src/components/Layout.tsx

```diff
@@ -0,0 +1,8 @@
+interface AppLayoutProps {
+  children: React.ReactNode;
+}
+
+function Layout({ children }: AppLayoutProps) {
+  return <div className="w-screen max-w-3xl m-auto">{children}</div>;
+}
+export default Layout;
```

Comment:
```suggestion
  return <div className="w-screen max-w-3xl m-auto">{children}</div>;
}

export default Layout;
```
export 는 줄바꿈 하는게 좋을 것 같습니다

Reply:
[5a07e68](https://github.com/sjoleee/bemyidol-frontend/pull/12/commits/5a07e680090d7622493c68e9516a23a0ea1e2675)에 반영했습니다